tutorials/clinical/10_ieeg_localize.py

```diff
@@ -173,42 +219,81 @@ def plot_overlay(image, compare, title, thresh=None):
 fig.tight_layout()
 del CT_data, T1
 
+# %%
+# Now we need to estimate the "head" coordinate transform.
+#
+# MNE stores digitization montages in a coordinate frame called "head"
+# defined by fiducial points (origin is halfway between the LPA and RPA
+# see :ref:`tut-source-alignment`). For sEEG, it is convenient to get an
+# estimate of the location of the fiducial points for the subject
+# using the Talairach transform (see :func:`mne.coreg.get_mni_fiducials`)
+# to use to define the coordinate frame so that we don't have to manually
+# identify their location.
+
+# create symbolic link to share ``subjects_dir``
+if not op.exists(op.join(subjects_dir, 'sample_seeg')):
+    os.symlink(op.join(misc_path, 'seeg', 'sample_seeg'),
+               op.join(subjects_dir, 'sample_seeg'))
```

Comment:
I don't like this. You alter the mne-sample-data folder. Why not fetching fsaverage in the sample_seeg folder?

Reply:
This was @larsoner 's suggested way of doing it

Reply:
why @larsoner ? I see this a side effects from a script

Reply:
Seems like a fairly harmless one. `fetch_fsaverage` to `mne-misc-data` directory also modifies / creates a side effect, just to that dataset instead of `sample`. This could be done, but it will use some hundreds of MBs and some download time rather than being instantaneous. I'm fine either way

Reply:
Can we just remove the `symlink` when we're done?

Reply:
hum... keep it like this then but I would suggest to do this really at the top of the file with a big warning that it's just for the demo here. You should not have to do this on your own data.

Reply:
I originally had a `subjects_dir` parameter for the template as well, perhaps we should just put that back

Reply:
Works for me